src/examples/config-file.ts

```diff
@@ -5,31 +5,33 @@ import {
 } from "@govtechsg/open-attestation/dist/types/__generated__/schema.2.0";
 import {
   IdentityProofType as IdentityProofTypeV3,
+  Method,
   ProofType,
   RevocationType as RevocationTypeV3,
-  Method,
 } from "@govtechsg/open-attestation/dist/types/__generated__/schema.3.0";
-import { walletReference } from "./wallet";
-import { formsV2, formsV3 } from "./forms";
 import { ConfigFileWithFormV2, ConfigFileWithFormV3 } from "../types";
+import { formsV2, formsV3 } from "./forms";
+import { walletReference } from "./wallet";
 
 export const configFileV2: ConfigFileWithFormV2 = {
   network: "ropsten", // do not change yet, oa-cli should hard overwrite with ropsten
   wallet: walletReference,
-  documentStorage: {
-    apiKey: "randomKey",
-    url: "https://api-ropsten.tradetrust.io/storage",
-  },
+  // remove document storage for the moment as it is only for ropsten test net.
+  // documentStorage: {
+  //   apiKey: "randomKey",
+  //   url: "https://api-ropsten.tradetrust.io/storage",
+  // },
```

Comment:
Will the endpoints work now that the fixtures in https://github.com/TradeTrust/tradetrust-website/pull/629 have been updated? I wondered because I see that the same addresses as the ones in the fixtures are also used here in: https://github.com/TradeTrust/tradetrust-config/blob/718822b454392b431c22c696e75b639352587423/src/generate-config.ts#L17-L18.

Reply:
from what i remember it wont work, need to double check the document storage, if i rmb correctly it somehow also checked for the network.

Reply:
no, document storage endpoints is another thing altogether, which reminds me we might need to re-setup that... 👀

https://github.com/Open-Attestation/oa-functions/blob/master/src/storage/create.ts#L30

Reply:
yeah need re setup for goerli network